packages/dashboard/src/lib/framework/extension-api/types/data-table.ts

```diff
@@ -43,7 +43,15 @@ export type BulkActionComponent<Item extends { id: string } & Record<string, any
  * @since 3.4.0
  */
 export type BulkAction = {
+    /**
+     * @description
+     * Optional order number to control the position of this bulk action in the dropdown.
```

Comment:
should add "a larger number will appear lower in the list" (double check that's the case 😆 )